homeassistant/components/flexit/climate.py

```diff
@@ -175,7 +177,7 @@ def target_temperature(self):
     @property
     def hvac_mode(self):
         """Return current operation ie. heat, cool, idle."""
-        return self._current_operation
+        return HVACMode.COOL
```

Comment:
I think we should remove this property as it is now replaced by `self._attr_hvac_action`

Reply:
Just realized this integration only supports cool mode and was using HVAC mode instead of action to represent the current state, so it should be OK to fix it to `HVACMode.COOL`

Reply:
Yeah, it's all a bit messy. It actually seems to be able to do both cooling and heating according to the potential actions, but setting it to HEAT_COOL allows setting a range.